### PR TITLE
NETOBSERV-1744: Kafka replicas update

### PR DESCRIPTION
**Pull request by Amoghrd**
PR has following changes:
- Update kafkaReplicas
- Make kafkaReplicas variable update values based on workload
- Remove references to router-perf and cluster-density

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -246,11 +250,10 @@ pipeline {
         )
         choice(
             name: 'WORKLOAD',
-            choices: ['None', 'cluster-density-v2', 'cluster-density', 'node-density-heavy', 'node-density', 'router-perf', 'ingress-perf'],
+            choices: ['None', 'cluster-density-v2', 'cluster-density', 'node-density-heavy', 'ingress-perf'],
```
Comment (memodi) [inline Jenkinsfile:253]:
```suggestion
            choices: ['None', 'cluster-density-v2', 'node-density-heavy', 'ingress-perf'],
```

File: Jenkinsfile [hunk 2]
```diff
@@ -229,10 +229,14 @@ pipeline {
...
-            defaultValue: '3',
+            defaultValue: '',
             description: '''
                 Replicas should be at least half the number of Kafka TOPIC_PARTITIONS and should not exceed number of TOPIC_PARTITIONS or number of nodes:<br/>
-                3 - default for non-perf testing environments<br/>
+                3 - for non-perf testing environments<br/>
+                6 - default for node-density-heavy workload<br/>
+                12 - default for ingress-perf workload<br/>
+                18 - default for cluster-density-v2 workload<br/>
+                If empty, job will automatically set value based on the workload run<br/>
```
Comment (memodi) [inline Jenkinsfile:239]:
```suggestion
                <b>Leave this empty if you're triggering standard perf-workloads, default of 6, 12 and 18 FLP_KAFKA_REPLICAS will be used for node-density-heavy, ingress-perf and cluster-density-v2 workloads respectively</b><br/>
                3 - for non-perf testing environments<br/>
                <b>Use this field to overwrite default FLP_KAFKA_REPLICAS</b><br/>
```

**Review by memodi — CHANGES_REQUESTED** [review]

**Comment by memodi** [comment]
@Amoghrd - can you trigger a test run on your branch for ndh to make sure it's working as expected? Also, please rebase this PR, I see several merge commits.

**Comment by Amoghrd** [comment]
Perf run failed but the params were passed correctly [job#15](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/aramesha-e2e-benchmarking-multibranch-pipeline/job/kafkaReplicas/15/console)

**Review by memodi — APPROVED** [review]
/lgtm
thanks @Amoghrd

**Pull request merged**